opencti-platform/opencti-front/src/private/components/events/stix_sighting_relationships/StixSightingRelationshipCreationFromEntity.jsx

```diff
@@ -455,29 +460,41 @@ const StixSightingRelationshipCreationFromEntity = ({
       </div>
     );
   };
+
+  const openElement = () => {
+    switch (variant) {
+      case 'controlledDial':
+        return CreateEntityControlledDial('entity_Sighting')({ onOpen: handleOpen });
```

Comment:
This is why I'm not fond of factories ; it makes the code unclear.
We can have a dedicated component like I said earlier.